app/json_schemas/customer_schema.rb

```diff
@@ -41,8 +41,8 @@ def self.address_example
       street_address_2: "",
       postal_code: "1234",
       locality: "Melbourne",
-      region: "Victoria",
-      country: "Australia",
+      region: { code: "Vic", name: "Victoria" },
+      country: { code: "AUS", name: "Australia" },
```

Comment:
The examples are:
```suggestion
      region: { code: "VIC", name: "Victoria" },
      country: { code: "AU", name: "Australia" },
```

Reply:
Okay, looks like `Vic` is in our code base and production database. It doesn't really matter but Australia Post usually wants `VIC` in the address. Let's keep that as is and change it if it's a problem. That wouldn't be related to the API but related to the production database. And maybe we should use case-insensitive search here.

Reply:
@jibees This is a very good example of me discovering that I was wrong and being able to fix it because I'm working on the issue myself. I'm very encouraged by the team work we've done here. :muscle: